x-pack/plugins/security_solution/public/common/components/alert_count_by_status/alert_count_by_rule_by_status.tsx

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+
+import type { EuiBasicTableColumn } from '@elastic/eui';
+import { EuiBasicTable, EuiEmptyPrompt, EuiLink, EuiPanel, EuiToolTip } from '@elastic/eui';
+import { euiStyled } from '@kbn/kibana-react-plugin/common';
+
+import type { Status } from '../../../../common/detection_engine/schemas/common';
+import { SecurityPageName } from '../../../../common/constants';
+import { useNavigateToTimeline } from '../../../overview/components/detection_response/hooks/use_navigate_to_timeline';
+import { useQueryToggle } from '../../containers/query_toggle';
+import { FormattedCount } from '../formatted_number';
+import { HeaderSection } from '../header_section';
+import { HoverVisibilityContainer } from '../hover_visibility_container';
+import { BUTTON_CLASS as INSPECT_BUTTON_CLASS } from '../inspect';
+import { LastUpdatedAt } from '../last_updated_at';
+import { SecuritySolutionLinkAnchor } from '../links';
+import { useLocalStorage } from '../local_storage';
+import { MultiSelectPopover } from './components';
+import * as i18n from './translations';
+import type { AlertCountByRuleByStatusItem } from './use_alert_count_by_rule_by_status';
+import { useAlertCountByRuleByStatus } from './use_alert_count_by_rule_by_status';
+
+interface EntityFilter {
+  field: string;
+  value: string;
+}
+interface AlertCountByStatusProps {
+  entityFilter: EntityFilter;
+  signalIndexName: string | null;
+}
+
+interface StatusSelection {
+  [fieldName: string]: Status[];
+}
+
+type GetTableColumns = (
+  openRuleInTimelineWithAdditionalFields: (ruleName: string) => void
+) => Array<EuiBasicTableColumn<AlertCountByRuleByStatusItem>>;
+
+const KIBANA_RULE_ALERT_FIELD = 'kibana.alert.rule.name';
+const STATUSES = ['open', 'acknowledged', 'closed'] as const;
+const ALERT_COUNT_BY_RULE_BY_STATUS = 'alerts-by-status-by-rule';
+const LOCAL_STORAGE_KEY = 'alertCountByFieldNameWidgetSettings';
+
+const StyledEuiPanel = euiStyled(EuiPanel)`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  overflow: hidden;
+  height: 308px;
```

Comment:
<img width="790" alt="Screen Shot 2022-09-20 at 10 34 48 AM" src="https://user-images.githubusercontent.com/6935300/191327276-27a4e686-7bf1-42f4-b8f0-3340bed046be.png">

I don't think you need `height` here, seems to work fine without it. `height` causes this ugly behavior when both are toggled off. Also pretty strange that both headings are "Alerts"


w/o `height`:
<img width="1099" alt="Screen Shot 2022-09-20 at 12 44 26 PM" src="https://user-images.githubusercontent.com/6935300/191327955-66174551-bbc2-4f83-870a-b3db30621caf.png">

Reply:
Good catch. In a past implementation I had to give it a height for it to align with something. I dont have that problem with this version though.

Giving it a max height though for scrollability reasons.

Reply:
I had a convo with Paul about the names too. And that too is updated 👌